tests/importable_test.py

```diff
@@ -19,8 +20,31 @@ def test_entry_points_exposed(entry_points_group: str) -> None:
     pre-installed.
     """
     entry_points = _discover_entry_points(group=entry_points_group)
-    assert 'x' in entry_points.names
+    expected_entry_points = {
+        'aim': 'aim_plugin',
```

Comment:
@chrismeyersfsu FYI such things should go into either parametrize or `pytest-subtest` so that they show up separately in the test report.